tests/components/lamarzocco/conftest.py

```diff
@@ -36,6 +36,8 @@ async def init_integration(
     hass: HomeAssistant, mock_config_entry: MockConfigEntry, mock_lamarzocco: MagicMock
 ) -> MockConfigEntry:
     """Set up the LaMetric integration for testing."""
+    if not hass.config_entries.async_get_entry(mock_config_entry.entry_id):
+        mock_config_entry.add_to_hass(hass)
```

Comment:
`init_integration` is already calling this, so you add it twice now. What’s the benefit?

Reply:
`2024-02-14 10:19:29.886 ERROR    MainThread homeassistant.config_entries:config_entries.py:1200 An entry with the id 857f041d2415c215c9ef2e243f5e2428 already exists`

Reply:
Actually its still being added twice

Reply:
fixed in c45fdcba16a677fd0772d989dce69ed83cb2c7a5